Assets/LeapMotion/Modules/InteractionEngine/Scripts/InteractionManager.cs

```diff
@@ -345,6 +345,14 @@ public class InteractionManager : MonoBehaviour, IInternalInteractionManager,
           // (This will clear softContacts and originalVelocities as well.)
           using (new ProfilerSample("Apply Soft Contacts")) {
             if (_softContacts.Count > 0) {
+              RuntimeGizmoDrawer drawer;
+              if (_drawControllerRuntimeGizmos && RuntimeGizmoManager.TryGetGizmoDrawer(out drawer)) {
```

Comment:
The InteractionManager is already an IRuntimeGizmoComponent, so it has an OnDrawRuntimeGizmos method; we should do all runtime gizmo drawing there so it's all in one place. I don't have anything against TryGetGizmoDrawer, but I don't want to mix the two gizmo drawing patterns.